features/features.go

```diff
@@ -18,7 +18,8 @@ import (
 type Features struct {
 	// DoSaves flags the feature to only save artifacts under directly referenced targets
 	// of those referenced by BUILDs
-	ReferencedSaveOnly bool `long:"referenced-save-only" description:"only save artifacts that are directly referenced"`
+	ReferencedSaveOnly     bool `long:"referenced-save-only" description:"only save artifacts that are directly referenced"`
+	UseCopyIncludePatterns bool `long:"use-copy-include-patterns" description:"specify an include pattern to buildkit when performing copies"`
```

Comment:
This isn't currently referenced anywhere else; however, I needed a second flag for unit testing the comma splitting.

Once this PR is in, i'll rebase https://github.com/earthly/earthly/pull/1085 against this code which will then make use of this field for feature flipping.